install_scripts.sh

```diff
@@ -37,6 +37,30 @@ file_changed_in_pr() {
   ) && return 0 || return 1
 }
 
+sed_update_if_changed() {
+    # Usage: sed_update_if_changed 's/foo/bar/' file.txt
+    if [ "$#" -ne 2 ]; then
+        echo "Usage: sed_update_if_changed 'sed_command' file" >&2
+        return 1
+    fi
+
+    local sed_command="$1"
+    local file="$2"
+    local tmp_file="$(mktemp "${file}.XXXXXX")"
+
+    sed "$sed_command" "$file" > "$tmp_file" || {
+        rm -f "$tmp_file"
+        echo "sed command failed" >&2
+        return 1
+    }
+
+    if ! diff -q "$file" "$tmp_file" > /dev/null; then
+        mv "$tmp_file" "$file"
```

Comment:
@ocaisa We need to be a bit more careful here w.r.t. permissions, how about:
```suggestion
        # don't use mv, we want to retain same file permissions
        cat "$tmp_file" > "$file"